src/main/java/cuchaz/enigma/gui/elements/MenuBar.java

```diff
@@ -258,21 +263,63 @@ public MenuBar(Gui gui) {
 						JFrame frame = new JFrame(I18n.translate("menu.view.languages.title"));
 						Container pane = frame.getContentPane();
 						pane.setLayout(new FlowLayout());
-						
+
 						JLabel text = new JLabel((I18n.translate("menu.view.languages.summary")));
 						text.setHorizontalAlignment(JLabel.CENTER);
 						pane.add(text);
-						
+
 						JButton okButton = new JButton(I18n.translate("menu.view.languages.ok"));
 						pane.add(okButton);
 						okButton.addActionListener(arg0 -> frame.dispose());
-						
+
 						frame.pack();
 						frame.setLocationRelativeTo(this.gui.getFrame());
 						frame.setVisible(true);
 					});
 				}
 
+				JMenu scale = new JMenu(I18n.translate("menu.view.scale"));
+				{
+					ButtonGroup scaleGroup = new ButtonGroup();
+					Map<Float, JRadioButtonMenuItem> map = IntStream.of(100, 125, 150, 175, 200)
+							.mapToObj(scaleFactor -> {
+								float realScaleFactor = scaleFactor / 100f;
+								JRadioButtonMenuItem menuItem = new JRadioButtonMenuItem(String.format("%d%%", scaleFactor));
+								menuItem.addActionListener(event -> ScaleUtil.setScaleFactor(realScaleFactor));
+								scaleGroup.add(menuItem);
+								scale.add(menuItem);
+								return new Pair<>(realScaleFactor, menuItem);
+							})
+							.collect(Collectors.toMap($ -> $.a, $ -> $.b));
```

Comment:
Don't use `$` in a variable name, names containing `$` are reserved to be internal.